src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2566,7 +2560,11 @@ namespace NServiceBus.Settings
     public class SettingsHolder : NServiceBus.Settings.ReadOnlySettings
     {
         public SettingsHolder() { }
+        [System.ObsoleteAttribute("Property injection is no longer supported, please use constructor injection inste" +
+            "ad. Will be removed in version 7.0.0.", true)]
```

Comment:
Any reason 'should be removed in' is put in the message? think it should be: 

```RemoveInVersion = "7.0", TreatAsErrorFromVersion = "6.0"```

Same applies to the usage below.